Documentation/edgefs-quickstart.md

```diff
@@ -0,0 +1,103 @@
+---
+title: EdgeFS Geo-Transparent Storage
+weight: 3
+indent: true
+---
+
+# EdgeFS Geo-Transparent Storage Quickstart
+
+This guide will walk you through the basic setup of a EdgeFS cluster namespaces and enable you to consume S3 object, NFS file access, and iSCSI block storage
+from other pods running in your cluster, in Geo-Transparent and distributed ways.
+
+## Minimum Version
+
+EdgeFS operator, CSI plugin and CRDs were tested with Kubernetes **v1.11** or higher.
+
+## Prerequisites
+
+To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [follow these instructions](k8s-pre-reqs.md).
+
+If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, make sure your host has at least 5GB of space available on the specified path.
+
+## TL;DR
+
+If you're feeling lucky, a simple EdgeFS Rook cluster can be created with the following kubectl commands. For the more detailed install, skip to the next section to [deploy the Rook operator](#deploy-the-rook-operator).
+```
+cd cluster/examples/kubernetes/edgefs
+kubectl create -f operator.yaml
+kubectl create -f cluster.yaml
+```
+
+After the cluster is running, you can create [NFS, S3, or iSCSI](#storage) storage to be consumed by other applications in your cluster.
+
+## Deploy the Rook Operator
+
+The first step is to deploy the Rook system components, which include the Rook agent running on each node in your cluster as well as Rook operator pod.
+
+EdgeFS Rook Operator needs cluster role binding in terms of to initialize certain configurations:
```

Comment:
any more details about this `cluster-admin` usage? what configurations need this, is it required for the mainline scenarios most people will want to do?

"cloud account email or id": will this attempt to integrate or authenticate with external cloud services? i'm a bit confused about the scope of access that is trying to be granted here. can you help me understand better?

Reply:
Good catch. Clarified - this is for gcloud users only.